app/src/main/java/com/brewthings/app/util/Calculations.kt

```diff
@@ -1,20 +1,27 @@
 package com.brewthings.app.util
 
 import com.brewthings.app.data.model.RaptPillData
-import com.brewthings.app.util.datetime.daysBetweenIgnoringTime
 
-fun calculateABV(og: Float, fg: Float, feedings: Float): Float {
-    if (og <= 1.0 || fg <= 1.0) return 0f
-    return (og.plus(feedings) - fg) * 131.25f
+fun calculateFeeding(previousGravity: Float, actualGravity: Float): Float =
+    actualGravity.minus(previousGravity)
+
+fun calculateABV(og: Float, fg: Float, feedings: List<Float>): Float? {
+    if (og <= 1.0 || fg <= 1.0) return null
+    return (og.sumAll(feedings) - fg) * 131.25f
```

Comment:
this is now the only formula in the app to calculate the ABV given of, fg and feedings.
will be useful when we will support more formulas, configurable from the settings, like MeadMate does.